src/verify.cc

```diff
@@ -123,8 +123,10 @@ Status verifyJwt(const Jwt& jwt, const Jwks& jwks, int64_t now) {
       // Verification succeeded.
       return Status::Ok;
     } else if ((jwk->pem_format_ || jwk->kty_ == "RSA") &&
-               verifySignatureRSA(jwk->evp_pkey_.get(), EVP_sha256(),
-                                  jwt.signature_, signed_data)) {
+        verifySignatureRSA(jwk->evp_pkey_.get(),
+                           EVP_sha256(),
+                           jwt.signature_,
+                           signed_data)) {
```

Comment:
Please use 
script/check-style

to format the code

Reply:
Fixed.